3 testing WriteLine for testing

ChristMagallon opened the pull request:
Testtuuu

ChristMagallon committed:
3 testing WriteLine for testing

Hermie03 commented on ConsoleApp1/Program.cs:
```diff
@@ -21,6 +21,9 @@ public void main()
         {
             do
             {
+                Console.WriteLine("Testttinggggg 123");
```
remove line 24 to 26

ChristMagallon committed:
done removing 24 to 26

ChristMagallon committed:
added another file for the other class

Hermie03 commented on ConsoleApp1/Program.cs:
```diff
@@ -12,9 +12,9 @@ static void Main(string[] args)
-        Account newAccount = new Account();
-        Showall newShowall = new Showall();
+    {
+        Showall NewShowAll = new Showall();
```
variable should have small letter in the beginning

Hermie03 commented on ConsoleApp1/ServerRoom.cs:
```diff
@@ -0,0 +1,101 @@
+
+namespace ConsoleApp1
+{
+   
```
remove whitespace

Hermie03 commented on ConsoleApp1/ServerRoom.cs:
```diff
@@ -0,0 +1,101 @@
+namespace ConsoleApp1
+{
+   
+        class Showall
```
make this public class ShowAll

Hermie03 commented on ConsoleApp1/ServerRoom.cs:
```diff
@@ -0,0 +1,101 @@
+            {
+                checker++;
+            }
+
```
Remove whitespace

Hermie03 commented on ConsoleApp1/ServerRoom.cs:
```diff
@@ -0,0 +1,101 @@
+                NewServer.setmiddleinitial(minitial);
+                NewServer.setyearlevel(yl);
+                NewServer.setcourse(course);
+                NewServer.checkerplusone();
```
Create a model for this:

`public class Student{
public string Name {get;set;}
public string MiddleInitial {get;set;}
public int YearLevel {get;set;}
public string Course {get;set;}
}`

Hermie03 reviewed (changes requested):
Create two folders on your project.

Folder name:
Models
Services

ChristMagallon committed:
change has been made

Hermie03 commented on ConsoleApp1/ServerRoom.cs:
```diff
@@ -0,0 +1,108 @@
+
+        public class Account
+        {
+            Server newServer = new Server();
```
Change this and use class Student

ChristMagallon committed:
Updated!
Note:
di ko kabalo unsaon pag call sa line 37 to 40
ni try ko " newStudent.setName(name); " and
"newStudent.Name(name);" both wont work

Hermie03 commented on ConsoleApp1/ServerRoom.cs:
```diff
@@ -0,0 +1,108 @@
+                newServer.setmiddleinitial(minitial);
+                newServer.setyearlevel(yl);
+                newServer.setcourse(course);
+                newServer.checkerplusone();
```
change this to newStudent.Name etc...hahaha

Hermie03 replied:
newStudent.setname = name;
                newStudent.setmiddleinitial = minitial;
                newStudent.setyearlevel = yl;
                newStudent.setcourse = course;

ChristMagallon replied:
awhhh lahi lagi and code sa kani

ChristMagallon replied:
abi nako kailangan pag brackets

Hermie03 replied:
setName(name) -> method ni siya hehehe
kanang NewStudent.Name = name; -> setters and getters ni siya, imagin lang nga ang table mao na si student unya ang naa sa table kay kanang name, course, level etc...

ChristMagallon replied:
ahhhwh orayt Noted!

ChristMagallon committed:
Updated
line 37 - 40

ChristMagallon committed:
sakto ba? di pa nako ma visualize ang lihok

ChristMagallon committed:
Final project. Updated

Note.
sige kog tanaw tutorial 10 best practices nya naka realise ko sa ako daghan sayop so ako gi fix tanan. for me mao ni ako final output.
